climada/entity/exposures/crop_production.py

```diff
@@ -541,62 +579,78 @@ def normalize_with_fao_cropyield(exp_firr, exp_noirr, input_dir=INPUT_DIR,
 
     fao_country = co.country_iso2faocode(country_list)
 
-    fao_yield = np.zeros(len(country_list))
+    fao_crop_production = np.zeros(len(country_list))
     ratio = np.ones(len(country_list))
-    exp_firr_norm = CropyieldIsimip()
-    exp_firr_norm = exp_firr
-    exp_noirr_norm = CropyieldIsimip()
-    exp_noirr_norm = exp_noirr
+    exp_firr_norm = CropProduction()
```

Comment:
Use

Reply:
You can use the copy module: 
`import copy`
`exp_firr_norm = copy.deepcopy(exp_firr)`